### PR TITLE
prepare release

### PR DESCRIPTION
**Pull request by folkertdev**

### CODE REVIEW

File: test-libbz2-rs-sys/Cargo.toml [hunk 1]
```diff
@@ -14,3 +14,7 @@ edition.workspace = true
 bzip2-sys = { version = "0.1.11", features = ["static"] }
 libbz2-rs-sys = { workspace = true, default-features = true, features = ["testing-prefix"] }
 libc.workspace = true
+
+[dev-dependencies]
+tempfile = "3.13.0"
+crc32fast = "=1.4.2"
```
Comment (bjorn3) [inline test-libbz2-rs-sys/Cargo.toml:20]: These don't seem to be used anywhere.

File: libbz2-rs-sys-cdylib/Cargo.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 [package]
 name = "libbz2-rs-sys-cdylib"
-version = "0.0.0"
+readme = "./README.md"
 edition = "2021"
-readme = "README.md"
 license = "bzip2-1.0.6"
 repository = "https://github.com/trifectatechfoundation/libbzip2-rs"
 homepage = "https://github.com/trifectatechfoundation/libbzip2-rs"
```
Comment (bjorn3) [inline libbz2-rs-sys-cdylib/Cargo.toml:7]: These can use workspace inheritance, right?

File: test-libbz2-rs-sys/Cargo.toml (hunk 1 above)
Comment (folkertdev) [reply test-libbz2-rs-sys/Cargo.toml:20]: they are actually, by `tests/quick.rs` and friends

File: libbz2-rs-sys-cdylib/Cargo.toml (hunk 2 above)
Comment (folkertdev) [reply libbz2-rs-sys-cdylib/Cargo.toml:6]: the cdylib has its own local readme (with instructions on cargo-c etc)

File: test-libbz2-rs-sys/Cargo.toml (hunk 1 above)
Comment (folkertdev) [reply test-libbz2-rs-sys/Cargo.toml:20]: ah but not here, right, I just discovered that too.

File: libbz2-rs-sys-cdylib/Cargo.toml (hunk 2 above)
Comment (folkertdev) [reply libbz2-rs-sys-cdylib/Cargo.toml:7]: also cdylib is not in the workspace for feature flag reasons (the defaults are different, and workspaces build the union of all features)

**Review by bjorn3 — APPROVED** [review]

**Pull request merged**